package.json

```diff
@@ -20,7 +20,7 @@
     "build::ts": "tsc -p ./tsconfig.json",
     "build::babel": "babel ./src --extensions .tsx,.ts,.js --out-dir ./dist",
     "clean": "rm -rf dist && mkdir dist",
-    "declarations": "ttsc --emitDeclarationOnly",
+    "declarations": "tsc --emitDeclarationOnly",
```

Comment:
Do you have link/docs related to what's changing here just for my education?

Reply:
We use the TTypescript to apply absolute path handling in the "build" script, which OG TS doesn't do.
Removing this dependency would mean we cannot use absolute paths in UIKit.

Reply:
If we wanted to use an alternative approach to this, in Arranger I prepatch TS itself, rather than using TTS.
That may solve whatever issues @ciaranschutte is finding in his local env.

Reply:
I see, and from their Github, it's deprecated. Thanks!

Reply:
oh, that bit I hadn't realised. we'll have to update UIKit to account for that (different scope, unless this is a blocker here)

Reply:
putting back in ttsc for build